modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -79,20 +69,12 @@ To customize the health probes:
 +
 For each of the probes, you can specify the request type - *HTTP GET*, *Container Command*, or *TCP Socket*,  from the drop-down list. The form changes as per the selected request type. You can then modify the default values for the other parameters, such as the success and failure thresholds for the probe, number of seconds before performing the first probe after the container starts, frequency of the probe, and the timeout value.
 
-
 Build Configuration and Deployment::
 Click the *Build Configuration* and *Deployment* links to see the respective configuration options. Some options are selected by default; you can customize them further by adding the necessary triggers and environment variables.
 +
 For serverless applications, the *Deployment* option is not displayed as the Knative configuration resource maintains the desired state for your deployment instead of a `DeploymentConfig` resource.
 
-Scaling::
-Click the *Scaling* link to define the number of pods or instances of the application you want to deploy initially.
-+
-For serverless applications, you can:
-
-* Set the upper and lower limit for the number of pods that can be set by the autoscaler. If the lower limit is not specified, it defaults to zero.
-* Define the soft limit for the required number of concurrent requests per instance of the application at a given time. It is the recommended configuration for autoscaling. If not specified, it takes the value specified in the cluster configuration.
-* Define the hard limit for the number of concurrent requests allowed per instance of the application at a given time. This is configured in the revision template. If not specified, it defaults to the value specified in the cluster configuration.
+include::snippets/serverless-scaling-odc.adoc[]
```

Comment:
Same concern as above. The "Scaling" link is not Serverless-specific. It exists also when a Deployment or DeploymentConfig is selected (in which case, it only allows setting of "replicas")

But with this change, we would make the Scaling snippet purely Serverless-specific:
```
Scaling:: Click the *Scaling* link to define autoscaling settings for serverless applications.
```
compared to the original
```
 Click the *Scaling* link to define the number of pods or instances of the application you want to deploy initially.
```

Reply:
I'll rename this one also. It doesn't need to be serverless specific because of the way these ODC docs are structured (we don't own them) we just need to make sure the options available for serverless are documented

Reply:
Made some updates to call out that some of these settings are just for serverless but scaling as a whole is not. It's tricky because we don't own these ODC docs and they're not really structured as I would like.